### PR TITLE
Update publish.sh

### PR DESCRIPTION
**Pull request by abhinavmpandey08**
*Issue #, if available:*

*Description of changes:*
Create new invalidation to invalidate cloudfront cache

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/publish.sh [hunk 1]
```diff
@@ -37,3 +37,4 @@ export AWS_CONFIG_FILE=$(pwd)/config
 export AWS_DEFAULT_PROFILE=release-prod
 
 aws s3 sync --acl public-read ./site s3://${BUCKET_NAME}
+aws cloudfront create-invalidation --distribution-id $DISTRIBUTION_ID --paths '/*'
```
Comment (micahhausler) [inline docs/publish.sh:40]: Can you  set `DISTRIBUTION_ID=${DISTRIBUTION_ID:-"E1G9R70424D2RN"}` above?

**Review by micahhausler — CHANGES_REQUESTED** [review]

File: docs/publish.sh (hunk 1 above)
Comment (micahhausler) [reply docs/publish.sh:40]: Can you also set the paths to 
```
/404.html /assets/ /community/ /index.html /search/ /sitemap.xml /sitemap.xml.gz/users/
```

**Comment by micahhausler** [comment]
/hold

Wait to unhold until permissions are updated

**Review by micahhausler — APPROVED** [review]
/lgtm
/approve

**Review by marceloboeira — APPROVED** [review]

**Pull request merged**